source/templates/actions.md

```diff
@@ -255,3 +255,41 @@ will trigger the action, but the user will remain on the current page.
 
 With `preventDefault=false`, if the user clicked on the link, Ember.js
 will trigger the action *and* the user will be directed to the new page.
+
+### Modifying the action's first parameter
+
+If a `value` option for the `{{action}}` helper is specified, its value will be
+considered a property path that will be read off of the first parameter of the
+action. This comes very handy with event listeners and enables to work with
+one-way bindings.
+
+```handlebars
+<label>What's your favorite band?</label>
+<input type="text" value={{favoriteBand}} onblur={{action "bandDidChange"}}/>
```

Comment:
Not totally, but I think it enables various things as the input value and the property (`favoriteBand`) are no longer two-way bound and thus you could do some filtering/transformation before you "write back" the value. This is silly but hopefully demonstrates my point:

```handlebars
<input type="text" value={{rating}} onblur={{action "clickRating" value="target.value"}}
```

```js
  actions: {
    clickRating(value) {
      const newValue = window.parseInt(value) + 1;
      this.set('rating', newValue);
    }
  }
```

When you focus out of the text field, the number you inputted will be incremented by one.